src/pages/TrackerPage.tsx

```diff
@@ -0,0 +1,143 @@
+  & > .seal {
+    margin: 0.25rem 0;
+    width: 1in;
+  }
+  & h2 {
+    margin-bottom: 0;
+  }
+  & h3 {
+    margin-top: 0;
+  }
+  & > .ballot-header-content {
+    flex: 1;
+    margin: 0 1rem;
+    max-width: 100%;
+  }
+`
+const Content = styled.div`
+  flex: 1;
+`
+
+const Highlight = styled.div`
+  background: #dddddd;
+  padding: 20px;
+`
+
+const Callout = styled.div`
+  float: left;
+  margin-right: 20px;
+  padding-right: 20px;
+  font-size: 2.5em;
+  font-weight: 700;
+`
+
+const Large = styled.div`
+  border-top: 0.2rem solid #000000;
+  padding: 20px;
+  font-size: 1.5em;
+`
+
+const TrackingID = styled.div`
+  border: 0.1rem solid #000000;
+  width: 50%;
+  padding: 20px;
+  font-size: 1.7em;
+`
+
+class TrackerPage extends React.Component<RouteComponentProps> {
+  public static contextType = BallotContext
+  public componentDidMount = () => {
+    window.addEventListener('afterprint', this.resetBallot)
+    window.print()
+  }
+  public componentWillUnmount = () => {
+    window.removeEventListener('afterprint', this.resetBallot)
+  }
+  public resetBallot = () => {
+    // Putting it back into event loop because infinite recursion otherwise.
+    // TODO: figure out a cleaner way to do this.
+    window.setTimeout(() => {
+      this.context.resetBallot('/cast')
+    }, 100)
+  }
+  public render() {
+    const {
+      election: { seal, title, county, state, date },
+    } = this.context
+
+    return (
+      <React.Fragment>
+        <Main>
+          <MainChild>
+            <Prose className="no-print">
+              <h1 aria-label="Print your tracker.">
+                Now printing your tracker...
+              </h1>
+              <p>Hold on just a few seconds.</p>
+            </Prose>
+            <div aria-hidden="true" className="print-only">
+              <Header>
+                <div
+                  className="seal"
+                  // TODO: Sanitize the SVG content: https://github.com/votingworks/bmd/issues/99
+                  dangerouslySetInnerHTML={{ __html: seal }} // eslint-disable-line react/no-danger
+                />
+                <Prose className="ballot-header-content">
+                  <h2>Track Your Ballot!</h2>
+                  <h3>{title}</h3>
+                  <p>
+                    {date}
+                    <br />
+                    {county}, {state}
+                  </p>
+                </Prose>
+              </Header>
+              <Content>
+                <Highlight>
+                  <Callout>&#8594;</Callout>
+                  <b>
+                    Use the unique tracking ID below to track your individual
+                    ballot and see it’s been recorded.
+                  </b>{' '}
+                  Although the contents of your ballot will always remain
+                  secret, you can verify that the vote recorded was at the time
+                  and place that you voted.
+                </Highlight>
+                <p />
```

Comment:
nit: if the `Highlight` component was a `p` element you wouldn't need to use this `<p />`

The `<Prose>` component is for styling hierarchical text elements. If all it's children are h1, h2, p, ul, table, etc. they will be nicely spaced.

Don't bother changing this. I'll give it a pass later.